core/src/main/java/org/apache/carbondata/core/util/path/CarbonTablePath.java

```diff
@@ -597,6 +599,27 @@ public static String getSegmentIdFromPath(String dataFileAbsolutePath) {
     }
   }
 
+  /**
```

Comment:
You can use getSegmentNumberFromSegmentFile method, instead of this. Please delete this.

Reply:
getSegmentNumberFromSegmentFile method is returning segmentId from segment file name. But i need segmentId from segmentFile absolute path. I have changed my method to make call to this method internally.

Reply:
you can compute the segmentFile name from the segmentFile absolute path in the test code itself. Or you can create a CarbonFile and then get the name in the test code

Reply:
If i write this logic of getting segmentFileName from segmentFile absolute path in test code, then i have to repeat the same code in both the testCases as i need segmentFileName in both testCases. Its better to write a method for it and call it to avoid repetition.